src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
```

Comment:
there are some suggestions to enhance readability and adhere to best practices. One suggestion is to store styles in a separate file or module instead of using inline styles. This will simplify the process of managing and reusing styles across the application.